### ISSUE TITLE
Future Lycanthrope and Vampire players

### ISSUE THREAD

**Issue by DrakeTheBeardie** [AlexModGuy/Ice-and-Fire-Suggestions#629]
One really interesting mod I had a decent chunk of fun within 1.7.10 was the Witchery mod. Not much the potion brewing, but the aspect that you could become a vampire or a werewolf. The Original Witchery never came to see 1.12.2, and to be honest, those two mechanics seemed heavily out of place within the mod.

However, this mod is specifically focused on the beasts of the wild mythos. I've seen that there are plans to add both Werewolves and Vampires; no doubt with a bunch of different interactions than Witchery ever did. My suggestion is to add in the feature that players are allowed to be infected and transform into Vampires and Werewolves. Either through random interaction or ritual. I feel it would add a new layer of player involvement with the mod that would have a bigger effect than just living alongside the beautiful creatures in this mod. 

To explain briefly how Witchery handled this feature; not to force you to adhere to it, but more to inform you of how it was done and how you can improve upon it, or do something different. Witchery allowed you to become either race by completing a ritual; both being moderately endgame for Vanilla. There was also the chance to be infected by other players on SMP servers. After this, there was a light quest system that unlocked abilities the further you progressed. Once you became level ten, you could infect other players. I found this an interesting system that provided a path to the abilities rather than spitting them out at the gate. Another important aspect of this feature is that it provided both buffs _and_ debuffs. As in, vampire players could no longer go outside during the day time or they would perish. Or how werewolves couldn't wear armor or hold weapons to make up for their increased unarmed damage and natural armor. Werewolves had the unique ability to use a moon charm to force transformations into either a wolf or a wolfman form. Now again, this isn't me telling you to adhere to these features, but merely something to think about how you can change, or make different. 

I know this sounds like me shilling for another mod, however, this is only one specific feature of Witchery and not the whole focus of the mod. A feature I really enjoyed and I feel the idea of infecting players with Lycanthropy and Vampirism would seat very well with this mod. Especially that it is canon in most mythologies that the curses of these demons can be spread. This could even extend to other species as well, such as the planned Wendigo, as if a player consumes enough rotten flesh, they could jumpstart a potential quest to become one. It's all in the eye of the beholder. 

I wanted to keep this less of a "me telling you exactly what features you should add and how to make them", and more of a "hey, this a cool idea you should consider, and if you don't want to do it, that's alright". Thanks for taking the time to read through my long wall of text; I appreciate it. This feels like such a unique mod, and I love the interesting features and would love to see something this cool get added too!

**TLDR:** Consider making it so the planned Werewolves and Vampires can infect players and turn them, similarly or dissimilarly to how Witchery did it. The idea can be extended to other species as well, like the Wendigo.

Sincerely, a fan of this mod~

**Comment by ThatOnePersonWho** [comment]
I believe that players will be able to become werewolves and vampires. However, players becoming werewolves will just have them turning into a glorified spectator that follows the werewolf AI around. No idea for vampires though.

**Comment by DwagonO** [comment]
I also want vamps added hell yes

**Comment by CeratoRaptor** [comment]
vampires are already confirmed mobs

**Comment by Hero-Artist13** [comment]
Yess please do this
